WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -273,7 +273,6 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 				EndRoundState.NotAllAlicesSign => "Aborted. Some Alices didn't sign. Go to blame round.",
 				EndRoundState.AbortedNotAllAlicesConfirmed => "Aborted. Some Alices didn't confirm.",
 				EndRoundState.AbortedLoadBalancing => "Aborted. Load balancing registrations.",
-				EndRoundState.AbortedDoubleSpendingDetected => "Aborted. Double spending detected.",
```

Comment:
Why remove this? Isn't it better if the client is aware of this, even if the backend returns a different abort reason until new release?

Reply:
As you can see in the previous commit `AbortedDoubleSpendingDetected` is now simply a synonym of  `AbortedNotAllAlicesConfirmed` and for that reason the switch statement cannot have it. It is the equivalent of having something like this:

```c#
var result = state switch {
    4 => "Ok",
    4 => "Good",
    4 => "Awesome"
}
```